NGCHM/WebContent/javascript/SummaryHeatMapDisplay.js

```diff
@@ -189,7 +189,7 @@ NgChm.SUM.summaryInit = function() {
 		mapName = mapName.substring(0,80) + "...";
 	}
 
-	nameDiv.innerHTML = "<b>NG-CHM Heat Map:</b>&nbsp;&nbsp;"+mapName;
+	nameDiv.innerHTML = "<b>Map Name:</b>&nbsp;&nbsp;"+mapName;
```

Comment:
Instead of the two nbsp's could you add the needed space with a css rule?

Reply:
The spacing is in the middle of the text, so I cannot apply a rule as a class directly on the DIV element (i.e. 2 spaces at beginning of Name).  I have created a double space and a tab CSS element that can be inserted into the middle of the text as a <SPAN> in lieu of the double &nbsp; statements.  However, for this case I think just replacing the &nbsp; with &ensp; (for double space) works better for code brevity.  As a larger task we could look at all the &nbsp;(s) in the code, there are many, and evaluate for the best solution given the context.